call_apply_bind_exercise/callApplyBind.js

```diff
@@ -0,0 +1,115 @@
+/*Fix the following code: 
+	var obj = {
+		fullName: "Harry Potter",
+		person: {
+			sayHi: function(){
+				return "This person's name is " + this.fullName
+			}
+		}
+	}
+*/
+
+var obj = {
+	fullName: "Harry Potter",
+	sayHi: function(){
+		return "This person's name is " + this.fullName
+	}
+}
+
+//console.log(obj.sayHi());
+
+/*List two examples of 'array-like objects':
+	arguments (keyword w/in functions)
+	document.getElementsByClassName() -- many DOM methods return 
+		array-like objects
+	
+*/
+
+/*Write a function called sumEvenArguments which takes all of the arguments 
+passed to a function and returns the sum of the even ones.*/
+
+function sumEvenArguments(...args){
+	return args.reduce(function(acc, el){
+		if (el % 2 === 0) acc += el
+		return acc;
+	}, 0);
+}
+
+sumEvenArguments(1,2,3,4) // 6
+sumEvenArguments(1,2,6) // 8
+sumEvenArguments(1,2) // 2
+
+/*Write a function called arrayFrom which converts an array-like-object 
+into an array.
+function sample(){
+	var arr = arrayFrom(arguments)
+	if(!arr.reduce) throw "This is not an array!"
+	return arr.reduce(function(acc,next){
+		return acc+next;
+	},0)
+}*/
+
+function arrayFrom(arrLike){
+	return [].slice.call(arrLike);
+}
+
+/*Write a function called invokeMax which accepts a function and a 
+maximum amount. invokeMax should return a function that when called 
+increments a counter. If the counter is greater than the maximum amount, 
+the inner function should return "Maxed Out"*/
+
+function invokeMax(fn, max){
+  var counter = 0;
+  return function(){
+	  var innerArgs = [].slice.call(arguments);
+	  if (counter < max) {
```

Comment:
It looks like you've got a mix of tabs and spaces here. It looks ok on github but ends up being weird on my editor.  I would try changing my settings to always do 2 spaces for a tab character (you can set that up with sublime).